modules/utxo-lib/src/networkTypes.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * @prettier
+ */
+
+/**
+ * @deprecated
+ */
+export const coins = {
+  BCH: 'bch',
+  BSV: 'bsv',
+  BTC: 'btc',
+  BTG: 'btg',
+  LTC: 'ltc',
+  ZEC: 'zec',
+  DASH: 'dash',
+} as const;
+
+/** @deprecated */
+export type CoinKey = keyof typeof coins;
+/** @deprecated */
+export type Coin = typeof coins[CoinKey];
+
+export type NetworkName =
+  | 'bitcoin'
+  | 'testnet'
+  | 'bitcoincash'
+  | 'bitcoincashTestnet'
+  | 'bitcoingold'
+  | 'bitcoingoldTestnet'
+  | 'bitcoinsv'
+  | 'bitcoinsvTestnet'
+  | 'dash'
+  | 'dashTest'
+  | 'litecoin'
+  | 'litecoinTest'
+  | 'zcash'
+  | 'zcashTest';
+
+export type Network = {
+  messagePrefix: string;
+  pubKeyHash: number;
+  scriptHash: number;
+  wif: number;
+  bip32: {
+    public: number;
+    private: number;
+  };
+  bech32?: string;
```

Comment:
Is this supposed to represent the bech32 prefix? Just not clear what it's being used for from the changes afaict.